source/reference/operator/query/mod.txt

```diff
@@ -117,3 +117,31 @@ operator with an array that contains four elements:
    	"$err" : "bad query: BadValue malformed mod, too many elements",
    	"code" : 16810
    }
+
+Modulo with Decimal Divisors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``modulo`` expression rounds down decimal input towards zero.
+
+The following examples demonstrates this behavior :
+   
+.. code-block:: javascript
```

Comment:
These examples are good, but this is not the most helpful format for users. Since this entire code block is copyable, users will expect to be able to copy and paste this block into their shell and have it run successfully. However, by including the input and output in the same code block, we run into issues.

One approach we could consider:

Place the input code block, followed by output code block, and label each code block accordingly. For example:
   
```
Input query:

.. code-block:: javascript

   db.inventory.find( { qty: { $mod: [ 4.0, 0 ] } } )

Results:

   .. code-block:: javascript
      :copyable: false
      
      { _id: 1, item: 'abc123', qty: 0 }
      { _id: 3, item: 'ijk123', qty: 12 }
```

And repeat this structure for each example.